android/app/src/main/java/com/microsoft/codepush/react/CodePush.java

```diff
@@ -236,6 +238,8 @@ private void initializeUpdateAfterRestart() {
                     // Therefore, deduce that it is a broken update and rollback.
                     rollbackPackage();
                 } else {
+                    // Clear the React dev bundle cache so that new updates can be loaded.
+                    clearReactDevBundleCache();
```

Comment:
@lostintangent BuildConfig is a generated class that is packaged to the parent project, e.g. com.mycompany.myapp.BuildConfig. We have no idea what their namespace is from our plugin package. One possible way is to get developers to pass in that value to CodePush via the constructor so that we can use it. That will involve a small but nonetheless breaking change to the current setup instructions. What do you think?